test/e2e/api_inheritance/api_inheritance_test.go

```diff
@@ -75,25 +76,32 @@ func TestAPIInheritance(t *testing.T) {
 				return
 			}
 
+			t.Logf("Bootstrapping workspace CRD in org workspace %q", testCase.logicalClusterName)
 			apiExtensionsClients, err := apiextensionsclient.NewClusterForConfig(cfg)
 			if err != nil {
 				t.Errorf("failed to construct apiextensions client for server: %v", err)
 				return
 			}
+			if err := config.BootstrapCustomResourceDefinitions(ctx, apiExtensionsClients.Cluster(testCase.logicalClusterName).ApiextensionsV1().CustomResourceDefinitions(), []metav1.GroupResource{
```

Comment:
@ncdc how did this work before? The non-"admin" workspace never had the workspace CRD installed (and should not until we have a controller doing that for org workspaces).

Reply:
This test used to only test Workspaces in admin.